CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+#### 0.7.4 (2020-08-20)
+
+#### Bug Fixes
+
+* remove `safe-eval` to fix `Sandbox Breakout / Arbitrary Code Execution` vulnerability ([831011a](https://github.com/aerogear/graphql-metadata/commit/831011ac7ffd87b20abd22797564822f291b024e))
```

Comment:
I wasn't sure whether to consider this a bug fix or not.

Reply:
Me neither :-(